addons/website_sale/tests/test_website_sale_visitor.py

```diff
@@ -66,11 +67,54 @@ def test_recently_viewed_company_changed(self):
         })
 
         self.website = self.website.with_user(public_user).with_context(website_id=self.website.id)
+
+        snippet_filter = self.env['website.snippet.filter'].search([('name', '=', 'Newest Products')])
```

Comment:
Not really critical but can't you have a more robust search than searching on name?
There can be duplicate by website? If yes, how is this finding the correct one? Why not searching by `action_server_id`?
If not, find by xml_id?

Reply:
Indeed safer, thanks!